identifiers/country-ca.csv

```diff
@@ -10775,490 +10780,490 @@ ocd-division/country:ca/province:on/ed:98,Welland,,,,,,,,,,,,,,
-ocd-division/country:ca/province:on/ed:middlesex_north,Middlesex North,,1917-10-05,,,,,,,,,,,,1867-08-06
-ocd-division/country:ca/province:on/ed:middlesex_south,Middlesex South,,1904-09-19,,,,,,,,,,,,1882-05-17
-ocd-division/country:ca/province:on/ed:middlesex_west,Middlesex West,,1968-04-22,,,,,,,,,,,,1867-08-06
-ocd-division/country:ca/province:on/ed:milton,Milton,,,,,,,,,,,,,,2015-08-02
-ocd-division/country:ca/province:on/ed:mississauga,Mississauga,,1979-03-25,,,,,,,,,,,,1973-12-05
-ocd-division/country:ca/province:on/ed:mississauga-brampton_south,Mississauga—Brampton South,,2015-08-01,,,,,,,,,,,,2004-05-23
-ocd-division/country:ca/province:on/ed:mississauga-erin_mills,Mississauga—Erin Mills,,,,,,,,,,,,,,2015-08-02
-ocd-division/country:ca/province:on/ed:mississauga-erindale,Mississauga—Erindale,,2015-08-01,,,,,,,,,,,,2004-05-23
-ocd-division/country:ca/province:on/ed:mississauga-lakeshore,Mississauga—Lakeshore,,,,,,,,,,,,,,2015-08-02
-ocd-division/country:ca/province:on/ed:mississauga-malton,Mississauga—Malton,,,,,,,,,,,,,,2015-08-02
-ocd-division/country:ca/province:on/ed:mississauga-streetsville,Mississauga—Streetsville,,,,,,,,,,,,,,2004-05-23
-ocd-division/country:ca/province:on/ed:mississauga_centre,Mississauga Centre,,2004-05-22,,,,,,,,,,,,1997-04-27
-ocd-division/country:ca/province:on/ed:mississauga_centre-2015,Mississauga Centre,,,,,,,,,,,,,,2015-08-02
-ocd-division/country:ca/province:on/ed:mississauga_east,Mississauga East,,2004-05-22,,,,,,,,,,,,1988-10-01
-ocd-division/country:ca/province:on/ed:mississauga_east-cooksville,Mississauga East—Cooksville,,,,,,,,,,,,,,2015-08-02
-ocd-division/country:ca/province:on/ed:mississauga_east-cooksville-2004,Mississauga East—Cooksville,,2015-08-01,,,,,,,,,,,,2004-05-23
-ocd-division/country:ca/province:on/ed:mississauga_north,Mississauga North,,1988-09-30,,,,,,,,,,,,1979-03-26
-ocd-division/country:ca/province:on/ed:mississauga_south,Mississauga South,,2015-08-01,,,,,,,,,,,,1979-03-26
-ocd-division/country:ca/province:on/ed:mississauga_west,Mississauga West,,2004-05-22,,,,,,,,,,,,1988-10-01
-ocd-division/country:ca/province:on/ed:monck,Monck,,1898-04-23,,,,,,,,,,,,1867-08-06
-ocd-division/country:ca/province:on/ed:muskoka,Muskoka,,1882-05-16,,,,,,,,,,,,1872-07-08
-ocd-division/country:ca/province:on/ed:muskoka-1904,Muskoka,,1925-09-04,,,,,,,,,,,,1904-09-20
-ocd-division/country:ca/province:on/ed:muskoka-ontario,Muskoka—Ontario,,1949-04-29,,,,,,,,,,,,1925-09-05
-ocd-division/country:ca/province:on/ed:muskoka_and_parry_sound,Muskoka and Parry Sound,,1904-09-19,,,,,,,,,,,,1882-05-17
-ocd-division/country:ca/province:on/ed:nepean,Nepean,,1997-04-26,,,,,,,,,,,,1988-10-01
-ocd-division/country:ca/province:on/ed:nepean-2015,Nepean,,,,,,,,,,,,,,2015-08-02
-ocd-division/country:ca/province:on/ed:nepean-carleton,Nepean—Carleton,,1988-09-30,,,,,,,,,,,,1979-03-26
-ocd-division/country:ca/province:on/ed:nepean-carleton-1997,Nepean—Carleton,,2015-08-01,,,,,,,,,,,,1997-04-27

```

Comment:
With this change most of the federal district OCD IDs that used `ed` prefix are completely dropped. I think it would be best to preserve them with `sameAs` aliases so that they map to their new `fed` counterpart.

Reply:
I don't think your suggestion is workable. #328 BROKE all provinces and territories by adding federal "ed" OCDIDs in the same space occupied by provincial "ed" OCDIDs. We cannot keep federal "ed" OCDIDs in that space. The only solution is to essentially revert and change to "fed". If we "sameAs" we just perpetuate the issue, because "sameAs" OCDIDs exist in the same space as non "sameAs" OCDIDs.

Reply:
The federal "ed" districts that hang directly off "country:ca" are untouched.

Reply:
To clarify is the issue that there are current collisions or that consumers of OCD IDs can no longer distinguish federal CA districts from provincial ones based solely on the OCD ID?

Either way I think it's fair to do this for the latter case, but I'd like to understand the issue better. 

I still have some concerns about systems that rely on OCD IDs not completely disappearing or later changing meaning. An alternate option for dealing with this could be to allow OCD IDs to be marked as invalid/deprecated. However I don't think we have an existing mechanism for that so it'd require an OCDEP.

Reply:
It’s at least the latter, and I think the former is also true.

Since the affected IDs haven’t been around that long (and since they weren’t that usable), I would just delete them.

Another option besides deprecation is versioning (also requires OCDEP / governance discussion). Current systems wouldn’t be aware of deprecation semantics, so they’d treat the deprecated OCDIDs as valid.

Reply:
Thanks, I think we can continue the discussion of how to handle situations like this in an issue. I'm ok with moving forward with this PR.